CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Change Log
 
+## [4.0.2](https://github.com/jazzband/django-silk/tree/4.0.2) (2020-08-07)
```

Comment:
`4.0.2` -> `4.1.0`

Reply:
Thank you for spotting that, it's fixed now :)